src/proto/Cosmos.proto

```diff
@@ -13,6 +13,11 @@ message Fee {
     uint64 gas = 2;
 }
 
+enum BroadcastMode {
+    BLOCK = 0;
+    ASYNC = 1;
```

Comment:
add `Sync` as `sync` and make it default

Reply:
have to change all old tests?, `block` is same as `sync`